Sources/eppo/UniversalFlagConfig.swift

```diff
@@ -28,13 +26,13 @@ public struct UniversalFlagConfig : Decodable {
         } catch let error as DecodingError {
             switch error {
             case .typeMismatch(_, let context):
-                throw UniversalFlagConfigError.parsingError("Type mismatch: \(context.debugDescription)")
+                throw UniversalFlagConfigError.parsingError("Type mismatch: \(context.debugDescription) - \(String(describing: context.underlyingError))")
```

Comment:
The existing error message does not provide enough information to diagnose the issue. the `underlyingError` should demonstrate the exact object that cannot be decoded.

* https://developer.apple.com/documentation/swift/decodingerror/context/underlyingerror